lang/en/texts/advanced-user-guide.html

```diff
@@ -0,0 +1,1015 @@
+<style type="text/css">
+  .simple-bold {
+    font-weight: bold;
+  }
+  .image {
+    margin-left: 0px;
+    margin-top: 0px;
+    transform: rotate(0rad) translateZ(0px);
+    -webkit-transform: rotate(0rad) translateZ(0px);
+  }
+</style>
+
+<img
+  alt="https://static.openfoodfacts.org/images/misc/openfoodfacts-logo-en-356x300.png"
```

Comment:
This will be available on OBF and OPFF, too. Do we want to hard-code the domain?

Reply:
Just deploying it manually on OFF. I think we need specific guides for OBF and OPFF.